### PR TITLE
Logging

### PR DESCRIPTION
**Pull request by achaikou**

### CODE REVIEW

File: python/docs/logging.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+In both cases, when occurred event is recoverable,
+`default python logging module <https://docs.python.org/3/library/logging.html#module-logging>`_
+is used to report it.
+
+Users have to
+`configure logging <https://docs.python.org/3/howto/logging.html#configuring-logging/>`_
+themselves.
+If user doesn't provide any,
+`default configuration <https://docs.python.org/3/howto/logging.html#what-happens-if-no-configuration-is-provided/>`_
+applies.
```
Comment (ErlendHaa) [inline python/docs/logging.rst:19]: This confuses me a bit. It's hard to tell which 'control' you are referring too. As this is the logger section, I would expect that the docs here refers to controlling the logging module. But it's actually talking about ErrorHandler. 

Would probably drop all talk of ErrorHandler, to limit the confusion, and simply talk about how dlisio used Pythons logger module, and how it should be configured. I think this should cover it:

> dlisio is using Python's standard library logging module to emit several logs. dlisio only supply the loggers, while [configuring](https://docs.python.org/3/howto/logging.html#configuring-logging) them is left as an exercise to the user. If not, the logging modules [default configuration](link) applies.

File: python/docs/logging.rst [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+For example, next may be done to make all modules to print to stderr info
+messages, not just warnings as it happens by default:
```
Comment (ErlendHaa) [inline python/docs/logging.rst:22]: > not just warnings as it happens by default

warnings and errors.

File: python/docs/logging.rst [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+.. code-block:: python
+
+    >>> import logging
```
[the remark this answers is not in the document]
Comment (ErlendHaa) [reply python/docs/logging.rst:21]: Maybe change the log levels to the same level to not confuse?

[the remark this answers is not in the document]
Comment (achaikou) [reply python/docs/logging.rst:21]: I did it on purpose.
- the official example has Debug level set both times. So by changing only one to Info would show what actually must be changed.
- Changing both to info in my opinion can confuse somewhat more as user might think both must be set on the same level.

Basically to use this correctly and without confusion user has to understand official logging documentation and this we can't help with...

**Review by ErlendHaa — APPROVED** [review]

**Pull request merged**